modules/serverless-admin-monitoring-eventing-channel.adoc

```diff
@@ -0,0 +1,28 @@
+[id="serverless-admin-monitoring-eventing-channel_{context}"]
+= Monitoring Knative Eventing channels
+
+You can use the {product-title} monitoring dashboards to view metrics for channels in your cluster.
+
+.Prerequisites
+
+* You have cluster administrator permissions, and access to the *Administrator* perspective in the {product-title} web console.
+* You have installed the {ServerlessOperatorName}, as well as the Knative Eventing component.
```

Comment:
For Kafka, one need to actually install the `KnativeKafka` CR onto the eventing namepace. You do not get Kafka, if "just" installing eventing